src/iblphotometry/preprocessing.py

```diff
@@ -1,67 +1,18 @@
 """
 This modules offers pre-processing for raw photometry data.
-It implements different kinds of pre-processings depending on the preference of the user.
-Where applicable, I have indicated a publication with the methods.
 """
-
 import scipy.signal
-import numpy as np
-
-import ibldsp.utils
-from iblutil.numerical import rcoeff
-
-import iblphotometry.helpers as helpers
-import pynapple as nap
-
 
-# def photobleaching_lowpass(raw_calcium: nap.Tsd, fs: float, **params):
-#     """
-#     Here the isosbestic recording is ignored, and the reference is computed as the low-pass component of the
-#     calcium band signal.
-#     :param calcium:
-#     :param params: dictionary with parameters
-#     {
-#         'butterworth_lowpass': {'N': 3, 'Wn': 0.01, 'btype': 'lowpass'}
-#         }
-#         dictionary with parameters for the butterworth filter applied to the calcium band for the sole purpose of regression
-#     :return:
-#     iso (np.array): the corrected isosbestic signal to be used as control
-#     ph (np.array): the corrected calcium signal
-#     """
-#     params = {} if params is None else params
-#     calcium_lp = utils.filt(
-#         raw_calcium,
-#         **params.get("butterworth_lowpass", {"N": 3, "Wn": 0.01, "btype": "lowpass"}),
-#     )
-#     # sos = scipy.signal.butter(
-#     #     fs=fs,
-#     #     output="sos",
-#     #     **params.get("butterworth_lowpass", {"N": 3, "Wn": 0.01, "btype": "lowpass"}),
-#     # )
-#     # calcium_lp = scipy.signal.sosfiltfilt(sos, raw_calcium)
-#     calcium = (raw_calcium.values - calcium_lp.values) / calcium_lp.values
-#     return nap.Tsd(t=raw_calcium.times(), d=calcium)
+def low_pass_filter(raw_signal ,fs):
+    params = {}
+    sos = scipy.signal.butter(fs=fs, output='sos', **params.get('butterworth_lowpass', {'N': 3, 'Wn': 0.01, 'btype': 'lowpass'}))
```

Comment:
There is a problem here it doesn't even read the params.

But is there really a need to wrap this in a function ?